src/pybind/mgr/dashboard_v2/frontend/src/app/shared/datatable/table/table.component.html

```diff
@@ -62,6 +62,7 @@
                  [footerHeight]="footer ? 'auto' : 0"
                  [limit]="limit > 0 ? limit : undefined"
                  [loadingIndicator]="loadingIndicator"
+                 [rowIdentity]="rowIdentity()"
```

Comment:
If we are using just 1 prop as the identity we should use `trackByProp`.

Reply:
Can't be used. It will loose track of the row if the table is updated. With `rowIdentity` the selection is still active after the update.